adapters/adgeneration/adgenerationtest/exemplary/single-banner.json

```diff
@@ -0,0 +1,151 @@
+                            "beaconurl": "https://tg.socdm.com/bc/v3?b=beaconID&t=.gif",
+                            "cpm": 46.6,
+                            "creativeid": "Dummy_supership.jp",
+                            "h": 250,
+                            "landing_url": "",
+                            "scheduleid": "512601",
+                            "trackers": {
+                                "imp": [
+                                    "https://tg.socdm.com/bc/v3?b=beaconID&t=.gif"
+                                ],
+                                "viewable_imp": [
+                                    "https://tg.socdm.com/aux/inview?creative_id=1093926&ctsv=CTSC&extra_field=ExtraField&family_id=512601&id=58278&loglocation_id=64919&lookupname=58278%3ASSPLOC%3A*&pos=SSPLOC&schedule_id=72115.76855.512601&seqid=SeqID&seqtime=SeqTime&xuid=XUID"
+                                ],
+                                "viewable_measured": [
+                                    "https://tg.socdm.com/aux/measured?creative_id=1093926&ctsv=CTSC&extra_field=ExtraField&family_id=512601&id=58278&loglocation_id=64919&lookupname=58278%3ASSPLOC%3A*&pos=SSPLOC&schedule_id=72115.76855.512601&seqid=SeqID&seqtime=SeqTime&xuid=XUID"
+                                ]
+                            },
+                            "ttl": 10,
+                            "w": 300,
+                            "weight": 1
+                        }
+                    ],
+                    "dealid": "test-deal-id",
+                    "rotation": "0",
+                    "scheduleid": "512601",
+                    "sdktype": "0",
+                    "trackers": {
+                        "imp": [
+                            "https://tg.socdm.com/bc/v3?b=beaconID&t=.gif"
+                        ],
+                        "viewable_imp": [
+                            "https://tg.socdm.com/aux/inview?creative_id=1093926&ctsv=CTSC&extra_field=ExtraField&family_id=512601&id=58278&loglocation_id=64919&lookupname=58278%3ASSPLOC%3A*&pos=SSPLOC&schedule_id=72115.76855.512601&seqid=SeqID&seqtime=SeqTime&xuid=XUID"
+                        ],
+                        "viewable_measured": [
+                            "https://tg.socdm.com/aux/measured?creative_id=1093926&ctsv=CTSC&extra_field=ExtraField&family_id=512601&id=58278&loglocation_id=64919&lookupname=58278%3ASSPLOC%3A*&pos=SSPLOC&schedule_id=72115.76855.512601&seqid=SeqID&seqtime=SeqTime&xuid=XUID"
+                        ]
+                    },
+                    "ttl": 10,
+                    "w": 300
+                }
+            }
+        }
+    ],
+    "expectedBidResponses": [
+        {
+          "currency": "JPY",
```

Comment:
Does this currency really expected ? 
Because go tests not checking for this value (you can change it to smth else and test still PASS). 
USD is the actual currency that was returned.

Reply:
I’m sorry. It is my mistake in the implementation. I will submit the PR later.

Reply:
I’m sorry. It is my mistake in the implementation. I will submit the PR later.

Reply:
I submitted a PR.
https://github.com/prebid/prebid-server/pull/1383